packages/flutter/lib/src/widgets/editable_text.dart

```diff
@@ -247,23 +247,17 @@ class EditableText extends StatefulWidget {
   EditableTextState createState() => new EditableTextState();
 
   @override
-  void debugFillDescription(List<String> description) {
-    super.debugFillDescription(description);
-    description.add('controller: $controller');
-    description.add('focusNode: $focusNode');
-    if (obscureText != false)
-      description.add('obscureText: $obscureText');
-    description.add('${style.toString().split("\n").join(", ")}');
-    if (textAlign != null)
-      description.add('$textAlign');
-    if (textScaleFactor != null)
-      description.add('textScaleFactor: $textScaleFactor');
-    if (maxLines != 1)
-      description.add('maxLines: $maxLines');
-    if (autofocus != false)
-      description.add('autofocus: $autofocus');
-    if (keyboardType != null)
-      description.add('keyboardType: $keyboardType');
+  void debugFillProperties(List<DiagnosticsNode> description) {
+    super.debugFillProperties(description);
+    description.add(new DiagnosticsProperty<TextEditingController>('controller', controller));
+    description.add(new DiagnosticsProperty<FocusNode>('focusNode', focusNode));
+    description.add(new DiagnosticsProperty<bool>('obscureText', obscureText, defaultValue: false));
+    style?.debugFillProperties(description);
```

Comment:
Yes. Also text style display varies across classes.
Notice here the text style properties aren't prefixed while for another case they are prefixed with the word `textStyle` and in another case prefixed with the word `text`
I think most consistent would be to add one extra line to the output and display output as
text
```
      ╘═╦══ text ═══
           ║ TextSpan:
           ║   TextStyle:
           ║     inherit: true
           ║     family: Ahem
           ║     size: 10.0
           ║     height: 1.0x
           ║   "12345"
           ╚═══════════
```
instead of
```
      ╘═╦══ text ═══
           ║ TextSpan:
           ║   inherit: true
           ║   family: Ahem
           ║   size: 10.0
           ║   height: 1.0x
           ║   "12345"
           ╚═══════════
```
In another existing case output looks something like this which I think is more verbose.
```
      ╘═╦══  ═══
           ║ TextSpan:
           ║   textStyle.inherit: true
           ║   textStyle.family: Ahem
           ║   textStyle.size: 10.0
           ║   textStyle.height: 1.0x
           ║   "12345"
           ╚═══════════
```

Search for
`textStyle?.debugFillProperties(description, prefix: 'textStyle.');`
to see where the second option occurs